plugin-niassa/src/main/java/ca/on/oicr/gsi/shesmu/niassa/NiassaServer.java

```diff
@@ -789,4 +790,9 @@ private void updateWorkflows() {
   public String url() {
     return url;
   }
+
+  public Optional<URI> vidarrUrl() {
+    String vidarrUrl = configuration.map(Configuration::getVidarrUrl).orElse(null);
+    return vidarrUrl != null ? Optional.of(URI.create(vidarrUrl)) : Optional.empty();
```

Comment:
More compactly:
```
return configuration.map(Configuration::getVidarrUrl).map(URI::create);
```
but I don't really care in this case since this isn't a maintenance concern.